controllers/flashsystemcluster_controller.go

```diff
@@ -416,15 +417,22 @@ func (r *FlashSystemClusterReconciler) ensureExporterDeployment(instance *odfv1a
 		r.Log.Error(err, "failed to create exporter deployment")
 		return err
 	}
-
+	newOwnerDetails := v1.OwnerReference{
```

Comment:
you can create newOwnerDetails once in the reconcile method and send it as a parameter to all sub-method below like ensureExporterDeployment,  ensureExporterService ... 

that way you won't duplicate code in all 3 methods

Reply:
Fixed